platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiActivitiesInSpaceTestIT.java

```diff
@@ -21,7 +21,7 @@
 
 @Tag("smoke")
```

Comment:
@Tag("smoke") allow authentification with root, @Tag("sniff") don't allow this now, we wait to create it